### PR TITLE
[TE] Return yaml errors after grouping and use consistent convention …

### PR DESCRIPTION
**Pull request by akshayrai**
…(subscription over notification)

### CODE REVIEW

File: thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/yaml/YamlResource.java [hunk 1]
```diff
@@ -405,10 +424,10 @@ long createSubscriptionGroup(String yamlAlertConfig) throws ValidationException
   }
 
   @POST
-  @Path("/notification")
+  @Path("/subscription")
```
Comment (jihaozh) [inline thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/yaml/YamlResource.java]: change this path requires frontend changes too.

File: thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/yaml/YamlResource.java [hunk 2]
```diff
@@ -483,10 +502,10 @@ void updateSubscriptionGroup(long oldAlertConfigID, String yamlAlertConfig) thro
   }
 
   @PUT
-  @Path("/notification/{id}")
+  @Path("/subscription/{id}")
```
Comment (jihaozh) [inline thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/yaml/YamlResource.java]: same as above.

File: thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/yaml/YamlResource.java (hunk 1 above)
Comment (akshayrai) [reply thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/yaml/YamlResource.java]: Yes, I have already communicated this with @aaronucsd

File: thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/yaml/YamlResource.java (hunk 2 above)
Comment (akshayrai) [reply thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/yaml/YamlResource.java]: as above

**Pull request merged**